src/components/ListPane.tsx

```diff
@@ -0,0 +1,42 @@
+import { IAlbum } from 'dxmp-common';
+import * as React from 'react';
+
+import './ListPane.scss';
+
+interface IListPaneProps {
+  albums: IAlbum[],
+};
+
+interface IListPaneState {
+  query: string,
+};
+
+export class ListPane extends React.Component<IListPaneProps> {
```

Comment:
I think the second argument to the `Component` generic should be `IListPaneState`.

Reply:
Whoops